lib/sonata/channelWeeklyAirdrop/calculateStakersReward.ts

```diff
@@ -0,0 +1,28 @@
+import { Staker } from "@/types/Stake";
+
+export default function calculateStakersReward(users: Staker[], totalAmount: number) {
+  const currentDate = new Date();
+
+  const userScores = users.map(user => {
+    let investedDays = 8;
```

Comment:
why is the default for `investedDays` 8?
- does this mean we default to labeling the time staked to `8 days`?
- would it make more sense for default to be `0` so someone without any previous stake gets `0` extra points? right now, would that person get `8 days` even though they just started staking today?